app/init.php

```diff
@@ -1066,8 +1075,28 @@ function (mixed $value) {
             ->getResource()
         ;
     }
+    $cache  = new Cache(new Sharding($adapters));
+
+    $client = new SyncOut('syncOut', new QueueRedis(App::getEnv('_APP_REDIS_HOST', 'redis'), App::getEnv('_APP_REDIS_PORT', '6379')));
```

Comment:
Multiple cache write events can happen in the source code. We said we should aggregate them in an event and only fire them on the API shutdown (if needed). This will fire a message to the queue on every write.